src/components/CTypePresentation/CTypePresentation.tsx

```diff
@@ -91,7 +91,7 @@ class CTypePresentation extends React.Component<Props, State> {
               size={size || DEFAULT_SIZE}
               theme="polkadot"
             />
-            {this.getLabel(cType.cType.metadata.title.default)}
+            {this.getLabel(cType.cType.schema.$id)}
```

Comment:
AFAICS, we have access to the metadata here, so lets just use it

Reply:
yes we do have access.